sklearn/metrics/cluster/__init__.py

```diff
@@ -19,12 +19,12 @@
 from .supervised import entropy
 from .unsupervised import silhouette_samples
 from .unsupervised import silhouette_score
-from .unsupervised import calinski_harabaz_score
+from .unsupervised import calinski_harabasz_score
 from .bicluster import consensus_score
 
 __all__ = ["adjusted_mutual_info_score", "normalized_mutual_info_score",
            "adjusted_rand_score", "completeness_score", "contingency_matrix",
            "expected_mutual_information", "homogeneity_completeness_v_measure",
            "homogeneity_score", "mutual_info_score", "v_measure_score",
            "fowlkes_mallows_score", "entropy", "silhouette_samples",
-           "silhouette_score", "calinski_harabaz_score", "consensus_score"]
+           "silhouette_score", "calinski_harabasz_score", "consensus_score"]
```

Comment:
During deprecation we should keep both the misspelled and corrected versions here